medusa/scene_exceptions.py

```diff
@@ -295,39 +279,31 @@ def _get_custom_exceptions(force):
 
     if force or should_refresh('custom_exceptions'):
         for indexer in indexerApi().indexers:
+            location = indexerApi(indexer).config['scene_loc']
+            logger.info(
+                'Checking for scene exception updates from {location}',
+                location=location
+            )
             try:
-                location = indexerApi(indexer).config['scene_loc']
-                logger.info(
-                    'Checking for scene exception updates from {location}',
-                    location=location
-                )
-
-                try:
-                    jdata = indexerApi(indexer).session.get(location, timeout=60).json()
-                except (ValueError, AttributeError, RequestException) as error:
-                    logger.debug(
-                        'Check scene exceptions update failed. Unable to '
-                        'update from {location}. Error: {error}'.format(
-                            location=location, error=error
-                        )
-                    )
-                    return custom_exceptions
-
-                indexer_ids = jdata[indexerApi(indexer).config['identifier']]
-                for indexer_id in indexer_ids:
-                    indexer_exceptions = indexer_ids[indexer_id]
-                    alias_list = [{exception: int(season)}
-                                  for season in indexer_exceptions
-                                  for exception in indexer_exceptions[season]]
-                    custom_exceptions[indexer][indexer_id] = alias_list
-            except Exception as error:
-                logger.error(
-                    'Unable to update scene exceptions for {indexer}.'
-                    ' Error: {error}'.format(
-                        indexer=indexer, error=error
+                # When any Medusa Safe session exception, session returns None and then AttributeError when json()
+                jdata = indexerApi(indexer).session.get(location, timeout=60).json()
```

Comment:
Actually I would create a new session for scene_exceptions. And you could use the MedusaSafeSession for that. It's just to get the scene_exceptions from cdn.pymedusa.com. Don't need to use the indexers sessions for that. You could rename xem_session to session. Change it to MedusaSafeSession. And use that for all the session calls in scene_exceptions.